websites/api-docs-smoke-test/src/@commercetools-docs/gatsby-theme-docs/overrides/page-header-side.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
-import GitHubIcon from '../../../icons/github.svg';
+import SvgGithub from '../../../icons/generated/Github';
```

Comment:
Don't know why I can't import it directly over `../icons` 🤔

Reply:
Fixed: https://github.com/commercetools/commercetools-docs-kit/pull/756/commits/c5975bbe8b5cd3a888af2de6d8f54d1d41a84fb8